src/lesson11/task2/Unique.java

```diff
@@ -0,0 +1,23 @@
+package lesson11.task2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Unique {
+    public static void main(String[] args) {
+        List<Integer> nums = new ArrayList<>();
+        Random r = new Random();
+        for (int i = 0; i < 10; i++) {
+            nums.add (r.nextInt(6));
+            }
+            System.out.println(nums);
+        List<Integer>uniqueNums = new ArrayList<>();
+        for (int num : nums){
+            if (nums.indexOf(num) ==nums.lastIndexOf(num)){
```

Comment:
We need to delete all numbers, which presents more than omce.
Will it work according to the example in task?

Reply:
Yes, it works
![11 2](https://github.com/KateWi/Home-work/assets/73164321/8bc8c58f-56d5-4513-adff-fae051f76e9f)